deploy/k8s/teardown.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+set -e
+cd "$(dirname "$0")"
+./check.sh
+
+if [ "$IGNORE_NOT_FOUND" = true ]; then
+    set +e
+fi
+
+echo "Deleting Control Plane resources..."
+kubectl get databases.cp.nuodb.com -o name | xargs -r kubectl delete
+kubectl get domains.cp.nuodb.com -o name | xargs -r kubectl delete
+kubectl get servicetiers.cp.nuodb.com -o name | xargs -r kubectl delete
+kubectl get helmfeatures.cp.nuodb.com -o name | xargs -r kubectl delete
+kubectl get databasequotas.cp.nuodb.com -o name | xargs -r kubectl delete
```

Comment:
Can you also add a line for `backups.cp.nuodb.com`?
Alternatively, you could use the same approach as in `nuodb-dbaas-deploy`:

```
for crd in $(kubectl get customresourcedefinitions --no-headers -o custom-columns=":metadata.name" | grep ".cp.nuodb.com"); do
    kubectl get $crd -o name | xargs -r kubectl delete --wait=false
done
for crd in $(kubectl get customresourcedefinitions --no-headers -o custom-columns=":metadata.name" | grep ".cp.nuodb.com"); do
    kubectl get $crd -o name | xargs -r kubectl delete
done
```

Reply:
Good idea.